routes/web.php

```diff
@@ -5,3 +5,7 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::get('/admin', function () {
+    return Admin::get(1);
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Call to static method get() on an unknown class Admin.